src/olympia/lib/settings_base.py

```diff
@@ -1357,6 +1357,11 @@ def get_db_config(environ_var, atomic_requests=True):
             'level': logging.WARNING,
             'propagate': False
         },
+        'z': {
+            'handlers': ['mozlog'],
+            'level': logging.INFO,
+            'propagate': True
```

Comment:
not ` 'propagate': False` like it was before all this?

Reply:
apparently not.. I can't explain why but it does not work without `propagate=True` 😕